CONTRIBUTING.md

```diff
@@ -92,6 +92,25 @@ _Before_ submitting a pull request, please make sure the following is done…
 
 1.  If you haven't already, complete the [CLA](https://code.facebook.com/cla/).
 
+#### Usage with Docker
```

Comment:
👍 Much better. For my eye keeping instructions without and with Docker separated makes them both more clear.